components/slices/TextBlock.vue

```diff
@@ -0,0 +1,29 @@
+<template>
```

Comment:
We also need to create a Slice component to render the Text slice.  Even though the Text slice is one of the slices that Prismic provides out of the box, our front end doesn't know anything about it or how to render it until we create the components to do so.